string id
uint8 score
